specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/stable/2022-01-01/managedapplication.json

```diff
@@ -0,0 +1,2726 @@
+        },
+        "servicePackageActivationMode": {
+          "type": "string",
+          "description": "The activation Mode of the service package",
+          "enum": [
+            "SharedProcess",
+            "ExclusiveProcess"
+          ],
+          "x-ms-enum": {
+            "name": "ServicePackageActivationMode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "SharedProcess",
+                "description": "Indicates the application package activation mode will use shared process."
+              },
+              {
+                "value": "ExclusiveProcess",
+                "description": "Indicates the application package activation mode will use exclusive process."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "ServiceResourcePropertiesBase": {
+      "type": "object",
+      "description": "The common service resource properties.",
+      "properties": {
+        "placementConstraints": {
+          "type": "string",
+          "description": "The placement constraints as a string. Placement constraints are boolean expressions on node properties and allow for restricting a service to particular nodes based on the service requirements. For example, to place a service on nodes where NodeType is blue specify the following: \"NodeColor == blue)\"."
+        },
+        "correlationScheme": {
+          "$ref": "#/definitions/CorrelationSchemeList"
+        },
+        "serviceLoadMetrics": {
+          "$ref": "#/definitions/ServiceLoadMetricsList"
+        },
+        "servicePlacementPolicies": {
+          "$ref": "#/definitions/ServicePlacementPoliciesList"
+        },
+        "defaultMoveCost": {
+          "$ref": "#/definitions/MoveCost"
+        },
+        "scalingPolicies": {
+          "$ref": "#/definitions/ScalingPolicyList"
+        }
+      }
+    },
+    "ServiceScalingMechanismKind": {
+      "type": "string",
+      "description": "Enumerates the ways that a service can be partitioned.",
+      "enum": [
+        "ScalePartitionInstanceCount",
+        "AddRemoveIncrementalNamedPartition"
+      ],
+      "x-ms-enum": {
+        "name": "ServiceScalingMechanismKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "ScalePartitionInstanceCount",
+            "description": "Represents a scaling mechanism for adding or removing instances of stateless service partition. The value is 0."
+          },
+          {
+            "value": "AddRemoveIncrementalNamedPartition",
+            "description": "Represents a scaling mechanism for adding or removing named partitions of a stateless service. The value is 1."
+          }
+        ]
+      }
+    },
+    "ServiceScalingTriggerKind": {
+      "type": "string",
+      "description": "Enumerates the ways that a service can be partitioned.",
+      "enum": [
+        "AveragePartitionLoadTrigger",
+        "AverageServiceLoadTrigger"
```

Comment:
Changing enum names is a breaking change. Please get a review from breaking change board as well.

Reply:
thanks, creating intake here: https://msazure.visualstudio.com/One/_workitems/edit/12968912